### ISSUE TITLE
clj repl window history gets out of sync

### ISSUE THREAD

**Issue by Gnurdle** [BetterThanTomorrow/calva#491]
one would expect that <alt><up-arrow> would always yield the last entered line.
In some cases, the like before is returned, and you can't get the last entered line
if you mess with it a time or two, you can *then* get the prior line
video attached (I hope)

[repl-trouble.zip](https://github.com/BetterThanTomorrow/calva/files/3907305/repl-trouble.zip)

**Comment by bpringe** [comment]
Hi @Gnurdle, I have a similar (or the same) issue on Ubuntu 18.04, though the history works fine on my Windows machine. What OS are you using?

**Comment by Gnurdle** [comment]
ubuntu 19.10, kde spin

**Comment by bpringe** [comment]
I discovered that on Linux only (ubuntu at least) the commands for historyUp and historyDown are simply being called twice. I also discovered that any commands mapped to "alt+<up-or-down-arrow>" get called twice. I cannot figure out why that is the case though. I don't sely e anything in my OS keyboard settings that point to a reason.

I mapped the historyUp command to something other than "alt+<up-or-down-arrow>" and it only got called once. Did a little searching online and can't find anything useful about why that might be happening with Linux. It could also be an issue related to VS Code in combination with Linux.
